relays/lib-substrate-relay/Cargo.toml

```diff
@@ -58,6 +58,4 @@ sp-runtime = { git = "https://github.com/paritytech/polkadot-sdk", branch = "mas
 [dev-dependencies]
 bp-rococo = { path = "../../chains/chain-rococo" }
```

Comment:
@serban300 can we also remove this? `lib-substrate-relay` should be generic, right?

Reply:
Oops, you're right. Thanks ! I'll open a follow-up PR.

Reply:
Done: https://github.com/paritytech/parity-bridges-common/pull/2900 . PTAL

Reply:
nice, thank you :)